libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -248,7 +248,7 @@ export class PickingComponent extends PickingListMixin(LitElement) {
     }
 
     return html`
-      <div class="picking-complete">
+      <div class="picking-complete" data-testid="picking-complete-placeholder">
```

Comment:
I'm not sure about these, because @tobi-or-not-tobi don't like them to appear in production code :) 

Let's remove them now (together with that cypress `findById` action) and have a discussion about selectors with everyone to come up with the final solution?

Reply:
Yup, let's remove it for now and discuss this later, I see the point of avoiding this.